internal/wazeroir/operations.go

```diff
@@ -798,7 +725,7 @@ func NewOperationBuiltinFunctionCheckExitCode() UnionOperation {
 }
 
 // Label is the label of each block in wazeroir where "block" consists of multiple operations,
-// and must end with branching operations (e.g. OperationBr or OperationBrIf).
+// and must end with branching operations (e.g. NewOperationBr or NewOperationBrIf).
 type Label struct {
```

Comment:
I believe this `Label` is no longer needed to be exported

Reply:
-> `label`, and updated a few tests that used that to use `NewLabelID()` instead. There are still a few usages of `label` but they should all be tests now.